packages/flutter/lib/src/widgets/pages.dart

```diff
@@ -56,7 +56,7 @@ class PageRouteBuilder<T> extends PageRoute<T> {
   /// Creates a route that delegates to builder callbacks.
   ///
   /// The [pageBuilder], [transitionsBuilder], [opaque], [barrierDismissible],
-  /// and [maintainState] arguments must not be null.
+  /// [maintainState] and [fullscreenDialog] arguments must not be null.
```

Comment:
nit: please use the Oxford comma.

```suggestion
  /// [maintainState], and [fullscreenDialog] arguments must not be null.
```